### ISSUE TITLE
Value cannot be null

### ISSUE THREAD

**Issue by duhruh** [geluk/pass-winmenu#50]
I'm using native-git, when trying to push or pull from the remote i get the following error

```
See the end of this message for details on invoking 
just-in-time (JIT) debugging instead of this dialog box.

************** Exception Text **************
System.ArgumentNullException: Value cannot be null.
Parameter name: name
   at LibGit2Sharp.RemoteCollection.get_Item(String name) in C:\projects\libgit2sharp\LibGit2Sharp\RemoteCollection.cs:line 39
   at PassWinmenu.ExternalPrograms.Git.Fetch()
```

**Comment by Baggykiin** [comment]
Hi, thanks for the bug report, a hard error like that is definitely not supposed to happen, so I'll look into that.

Pass-winmenu will always try to work with the branch that you're currently on. Is it possible that the branch you're on does not have a remote configured? (i.e. running `git push` without any additional arguments will fail due to a missing remote) If that is the case, you can resolve the issue for now by setting a remote for the branch you're on.

**Comment by duhruh** [comment]
@Baggykiin thanks for getting back to me! Yea when i click the option `Open Shell` from the task bar i can just run a `git push` and it pushes fine.

**Comment by duhruh** [comment]
If it helps here are my specs:

* pass-winmenu v1.9.1
* git version 2.12.1.windows.1
* gpg4win 2.3.4
* gpg 2.0.30
* Windows 10 Pro 64-bit

**Comment by geluk** [comment]
This should be fixed in v1.10. Let me know if that's not the case, and I'll reopen this.
